### ISSUE TITLE
Broken for border properties

### ISSUE THREAD

**Issue by jpnelson** [ben-eb/postcss-merge-longhand#21]
```
border-top-width: 1px;
  border-right-width: 1px;
  border-bottom-width: 1px;
  border-left-width: 1px;
```

is not the same as

```
border: 1px;
```

as this could remove other border properties that are set.

**Comment by joscha** [comment]
probably duplicate of #20.

**Comment by ben-eb** [comment]
OK, no problem; I know that the ideal result is `border-width:1px`, but it won't be fixed in this module as it is being refactored out of version 4. In its place I'm planning individual modules for property normalization/merging.

For now you can disable the module if it is causing any issues (or use the shorthand way of defining this).

**Comment by joscha** [comment]
is there docs on how to disable it?

**Comment by jpnelson** [comment]
we're using gulp-cssnano,

```
gulpCssnano({mergeLonghand: false})
```

is sufficient

**Comment by ben-eb** [comment]
Yep, for future reference the transforms are documented here too. http://cssnano.co/options/

**Comment by seancurtis** [comment]
Commented on the CSS Nano repo too, but including here to show the effect.

Current behaviour (too generic): http://jsbin.com/bakupi/edit?html,css,output
Expected behaviour: http://jsbin.com/bakupi/edit?html,css,output

**Comment by ben-eb** [comment]
Resolved in [3.0](https://github.com/ben-eb/postcss-merge-longhand/commit/947185b1b19549410f943e2c24184e809746e231).
